src/Fabulous/Memo.fs

```diff
@@ -75,7 +75,7 @@ module Memo =
     let private widgetDefinition: WidgetDefinition =
```

Comment:
Readd comment on top:
`// Memo isn't allowed in lists, so this will never get called`